packages/blockchain-tools/src/interfaces/MetamaskClient.ts

```diff
@@ -412,7 +412,7 @@ export default class MetaMaskWallet extends ChainBaseClass {
     }
     console.log("GAS", Number.parseInt(gasPrice), gasPrice)
     const gas = numHex(210000)
-    const wei = numHex(amount * 10 ** 18)
+    const wei = numHex(amount * 10 ** 18) // use chain decimals
```

Comment:
Good idea, would you like to set that up? Shouldn't be too hard, just use the network and chain info that's set up in the constructor.

Reply:
Done